cli/tools/coverage/mod.rs

```diff
@@ -452,6 +452,10 @@ fn filter_coverages(
   let exclude: Vec<Regex> =
     exclude.iter().map(|e| Regex::new(e).unwrap()).collect();
 
+  // Matches virtual file paths for doc testing
+  // e.g. file:///path/to/mod.ts$23-29.ts
+  let doc_test_re = Regex::new(r"\$\d+-\d+\.(ts|js)$").unwrap();
```

Comment:
@magurotuna Do you know what are the possible file extensions from doc example extraction?

Reply:
The extension is taken from the code fence, like:

````markdown
```ts
// becomes .ts
```

```tsx
// becomes .tsx
```
````

Possible extensions are listed here
https://deno-docs--magurotuna-improved-doc-te.deno.dev/runtime/fundamentals/testing/#documentation-tests

- js
- javascript
- mjs
- cjs
- jsx
- ts
- typescript
- mts
- cts
- tsx

Reply:
Thanks!